lib/buffer.js

```diff
@@ -505,7 +505,7 @@ Buffer.prototype.write = function(string, offset, length, encoding) {
 
   // XXX legacy write(string, encoding, offset, length) - remove in v0.13
   } else {
-    writeWarned = internalUtil.printDeprecationMessage(writeMsg, writeWarned);
+    writeWarned = internalUtil.deprecate(writeMsg, writeWarned);
```

Comment:
@vkurchatkin What do you think about introducing a small function just for printing deprecation messages for modules?